src/ValitronValidator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Validate;
+
+use Valitron\Validator as OriginalValidator;
+
+/**
+ * Valitron Validation Class with required fixes.
+ */
+class ValitronValidator extends OriginalValidator
+{
+    #[\Override]
+    protected function validateDate($field, $value)
```

Comment:
for each method add regular unit test matching the class (`ValitronValidatorTest.php`) and method `testValidateDate()` testing the fixed behaviour